SDWebImage/SDImageLoader.m

```diff
@@ -47,13 +47,13 @@
         }
     }
     if (!image) {
-        SDImageCoderOptions *options = @{SDImageCoderDecodeFirstFrameOnly : @(decodeFirstFrame), SDImageCoderDecodeScaleFactor : @(scale)};
+        SDImageCoderOptions *coderOptions = @{SDImageCoderDecodeFirstFrameOnly : @(decodeFirstFrame), SDImageCoderDecodeScaleFactor : @(scale)};
```

Comment:
This change is adoptable. Because this may shadow the `options` arg from the input arg. But this is not related to this PR.

For each PRs, it should contains the most related things. Don't try to bundle something unrelated to a single PR, because this will make it hard to review, and may cause issue if we want to revert.

Reply:
I agree with this approach as a generic rule of thumb, but I think we can let this one slip :)

Reply:
@dreampiggy 👌 I just think it's trivial, so I do not create a separate `PR`. I'll be careful next time. 😂